app/controllers/registrations_controller.rb

```diff
@@ -1,7 +1,28 @@
 class RegistrationsController < Devise::RegistrationsController
+
+  def new
+    build_resource({})
+    set_minimum_password_length
+    yield resource if block_given?
+    self.resource.build_customer
+    respond_with self.resource
+  end
+
   protected
 
   def after_sign_up_path_for(*)
     new_car_path
   end
+
+  private
+
+  def sign_up_params
+    params.require(resource_name)
+      .permit(:email,
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align `.permit` with `.require` on line 20.